cmd/posix.go

```diff
@@ -166,7 +180,7 @@ func (s *posix) checkDiskFree() (err error) {
 	// Allow for the available disk to be separately validate and we will validate inodes only if
 	// total inodes are provided by the underlying filesystem.
 	if di.Files != 0 {
-		availableFiles := 100 * float64(di.Ffree) / float64(di.Files)
+		availableFiles := float64(di.Ffree) * 0.95
```

Comment:
i see the comment : `// Remove 5% from free space for cumulative disk space used for journalling, inodes etc.`

i think you can increase `fsMinFreeInodes   = 10000 + 2` or ignore the 5% for journaling and inodes to keep the code simple